From 7b26846119617963303fad22ff4a1a4cb6fac924 Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Mon, 28 Nov 2011 22:41:12 +0900
Subject: [PATCH 2/5] copyright-format: only one exception per license, for
 additional permissions.

document that only one exception is allowed per license in a license
specification, and also that exception syntax may not be used when
additional restrictions are imposed, only when additional permissions are
being granted.

Wording: Steve Langasek <vorlon@debian.org>
Seconded: Ben Finney <ben+debian@benfinney.id.au>
Seconded: Craig Small <csmall@debian.org>
Closes: #633797
---
 copyright-format/copyright-format.xml |   16 ++++++++++++++--
 1 files changed, 14 insertions(+), 2 deletions(-)

diff --git a/copyright-format/copyright-format.xml b/copyright-format/copyright-format.xml
index 1cf1acf..c9b5568 100644
--- a/copyright-format/copyright-format.xml
+++ b/copyright-format/copyright-format.xml
@@ -885,10 +885,22 @@ License: MPL-1.1
         matches.
       </para>
       <para>
-        Exceptions and clarifications are signaled in plain text, by appending
+        An exception or clarification to a license is signaled in plain text, by appending
         <literal>with <varname><replaceable>keywords</replaceable></varname>
         exception</literal> to the short name.  This document provides a list of
-        keywords that refer to the most frequent exceptions.
+        keywords that must be used when referring to the most frequent
+        exceptions.  When exceptions other than these are in effect that modify
+        a common license by granting additional permissions, you may use an
+        arbitrary keyword not taken from the below list of keywords.  When a
+        license differs from a common license because of added restrictions
+        rather than because of added permissions, a distinct short name should
+        be used instead of <literal>with
+        <varname><replaceable>keywords</replaceable></varname>exception</literal>.
+      </para>
+      <para>
+        Only one exception may be specified for each license within a given
+        license specification.  If more than one exception applies to a single
+        license, an arbitrary short name must be used instead.
       </para>
       <para>
         The GPL <literal>Font</literal> exception refers to the text added to the
-- 
1.7.5.4

